src/test/java/org/ays/auth/controller/AdminRegistrationApplicationEndToEndTest.java

```diff
@@ -355,7 +358,7 @@ void givenValidAdminRegistrationApplicationCompleteRequest_whenAdminRegistered_t
         // Then
         String endpoint = BASE_PATH.concat("/admin-registration-application/").concat(applicationId).concat("/complete");
         MockHttpServletRequestBuilder mockHttpServletRequestBuilder = AysMockMvcRequestBuilders
-                .post(endpoint, completeRequest);
+                .post(endpoint, superAdminToken.getAccessToken(), completeRequest);
```

Comment:
Public endpointlerde token göndermiyoruz
```suggestion
                .post(endpoint, completeRequest);
```